web_src/js/markup/tasklist.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Attaches `change` handlers to markdown rendered tasklist checkboxes in comments.
+ * When a checkbox value changes, the corresponding [ ] or [x] in the markdown string is set accordingly and sent to the server.
+ * On success it updates the raw-content on error it resets the checkbox to its original value.
+ */
+export default function initMarkupTasklist() {
+  $(`.render-content.markup[data-can-edit='true']`).parent().each((_, container) => {
```

Comment:
```suggestion
  $(`.render-content.markup.can-edit`).parent().each((_, container) => {
```

Reply:
No strong preference on these from me. Data attribute is kind of more safe from accidential re-use in future features.